modules/n1ql/pages/n1ql-language-reference/altersequence.adoc

```diff
@@ -0,0 +1,418 @@
+= ALTER SEQUENCE
+:page-topic-type: reference
+:page-status: Couchbase Server 7.6
+:imagesdir: ../../assets/images
+:description: The ALTER SEQUENCE statement enables you to alter an existing sequence in a given scope.
+
+// Cross-references
+:authorization-overview: xref:learn:security/authorization-overview.adoc
+:query-context: xref:n1ql:n1ql-intro/queriesandresults.adoc#query-context
+:identifiers: xref:n1ql-language-reference/identifiers.adoc
+:logical-hierarchy: xref:n1ql-intro/sysinfo.adoc#logical-hierarchy
+
+// TEMP
+include::partial$n1ql-language-reference/horizontal-style.adoc[]
+
+[abstract]
+{description}
+
+== Purpose
+
+include::./sequenceops.adoc[tags=overview;attributes]
+
```

Comment:
nit: I would xref to the CREATE statement rather than repeating all of the info here, through Storage section.  
(if you only provide the detail in CREATE, can then include xrefs from there to alter etc.)

Reply:
Still thinking about this myself. Left as is for now but may reconsider in future.